tests/TestingUtils/Library.fs

```diff
@@ -45,41 +45,39 @@ module Result =
         | Ok m -> m
         | Error m -> m
 
-
-
 /// Fable compatible Expecto/Mocha unification
 module Expect = 
     open Utils
 
     let inline equal actual expected message = Expect.equal actual expected message
-    let inline notEqual actual expected message = Expect.notEqual actual expected message
```

Comment:
Why? This caused errors for me previously.

Reply:
In js

Reply:
inline decreases performance, and it is only required for equal.